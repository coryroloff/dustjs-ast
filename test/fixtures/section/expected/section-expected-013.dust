{#section:change.the.context}{/section}