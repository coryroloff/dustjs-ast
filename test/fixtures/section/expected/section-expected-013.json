{
	"type": "Template",
	"body": [
		{
			"type": "Section",
			"body": [
				{
					"type": "Block",
					"body": [
						{
							"type": "Buffer",
							"text": "",
							"loc": {
								"start": {
									"line": 1,
									"column": 29
								},
								"end": {
									"line": 1,
									"column": 29
								}
							}
						}
					],
					"loc": {
						"start": {
							"line": 1,
							"column": 29
						},
						"end": {
							"line": 1,
							"column": 29
						}
					}
				}
			],
			"params": [],
			"kind": "#",
			"id": {
				"type": "Identifier",
				"path": [
					{
						"type": "Key",
						"name": "section",
						"loc": {
							"start": {
								"line": 1,
								"column": 2
							},
							"end": {
								"line": 1,
								"column": 9
							}
						}
					}
				],
				"localContext": false,
				"loc": {
					"start": {
						"line": 1,
						"column": 2
					},
					"end": {
						"line": 1,
						"column": 9
					}
				}
			},
			"context": {
				"type": "Identifier",
				"path": [
					{
						"type": "Key",
						"name": "change",
						"loc": {
							"start": {
								"line": 1,
								"column": 10
							},
							"end": {
								"line": 1,
								"column": 16
							}
						}
					},
					{
						"type": "Key",
						"name": "the",
						"loc": {
							"start": {
								"line": 1,
								"column": 17
							},
							"end": {
								"line": 1,
								"column": 20
							}
						}
					},
					{
						"type": "Key",
						"name": "context",
						"loc": {
							"start": {
								"line": 1,
								"column": 21
							},
							"end": {
								"line": 1,
								"column": 28
							}
						}
					}
				],
				"localContext": false,
				"loc": {
					"start": {
						"line": 1,
						"column": 10
					},
					"end": {
						"line": 1,
						"column": 28
					}
				}
			},
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 39
				}
			}
		}
	],
	"tokens": [
		{
			"type": "Punctuator",
			"value": "{",
			"loc": {
				"start": {
					"line": 1,
					"column": 0
				},
				"end": {
					"line": 1,
					"column": 1
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "#",
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 2
				}
			}
		},
		{
			"type": "Key",
			"value": "section",
			"loc": {
				"start": {
					"line": 1,
					"column": 2
				},
				"end": {
					"line": 1,
					"column": 9
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ":",
			"loc": {
				"start": {
					"line": 1,
					"column": 9
				},
				"end": {
					"line": 1,
					"column": 10
				}
			}
		},
		{
			"type": "Key",
			"value": "change",
			"loc": {
				"start": {
					"line": 1,
					"column": 10
				},
				"end": {
					"line": 1,
					"column": 16
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 16
				},
				"end": {
					"line": 1,
					"column": 17
				}
			}
		},
		{
			"type": "Key",
			"value": "the",
			"loc": {
				"start": {
					"line": 1,
					"column": 17
				},
				"end": {
					"line": 1,
					"column": 20
				}
			}
		},
		{
			"type": "Punctuator",
			"value": ".",
			"loc": {
				"start": {
					"line": 1,
					"column": 20
				},
				"end": {
					"line": 1,
					"column": 21
				}
			}
		},
		{
			"type": "Key",
			"value": "context",
			"loc": {
				"start": {
					"line": 1,
					"column": 21
				},
				"end": {
					"line": 1,
					"column": 28
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"loc": {
				"start": {
					"line": 1,
					"column": 28
				},
				"end": {
					"line": 1,
					"column": 29
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "{",
			"loc": {
				"start": {
					"line": 1,
					"column": 29
				},
				"end": {
					"line": 1,
					"column": 30
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "/",
			"loc": {
				"start": {
					"line": 1,
					"column": 30
				},
				"end": {
					"line": 1,
					"column": 31
				}
			}
		},
		{
			"type": "Key",
			"value": "section",
			"loc": {
				"start": {
					"line": 1,
					"column": 31
				},
				"end": {
					"line": 1,
					"column": 38
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"loc": {
				"start": {
					"line": 1,
					"column": 38
				},
				"end": {
					"line": 1,
					"column": 39
				}
			}
		}
	],
	"loc": {
		"start": {
			"line": 1,
			"column": 0
		},
		"end": {
			"line": 1,
			"column": 39
		}
	}
}